PhysicalTherapy/PhyscialTherapy/Controllers/RoutineExerciseController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PhysicalTherapy.Models;
+using PhysicalTherapy.Repositories;
+
+namespace PhysicalTherapy.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/RoutineExercises")]
+    public class RoutineExerciseController : ControllerBase
+    {
+        private readonly IRoutineExerciseRepository _routineExerciseRepository;
+
+        public RoutineExerciseController(IRoutineExerciseRepository routineExerciseRepository)
+        {
+            _routineExerciseRepository = routineExerciseRepository;
+        }
+
+        [HttpGet]
+        [Produces(typeof(DbSet<RoutineExercise>))]
+        public IActionResult GetRoutineExercises()
+        {
+            var test = new ObjectResult(_routineExerciseRepository.GetAll());
```

Comment:
me doing var test was just me debuggin the object to be return. dont really need this part.

Reply:
I'll just merge and remove on one of my next branches.